src/pages/About/About.jsx

```diff
@@ -0,0 +1,59 @@
+import TeamMember from "@/components/TeamMember";
+import { useState } from "react";
+import { i18n, useTranslation } from "next-i18next";
+import { serverSideTranslations } from "next-i18next/serverSideTranslations";
+import Image from "next/image";
+
+import OurFounding from "public/OurFoundingPayan.png";
+
+import PageIntro from "@/components/PageIntro";
+import Layout from "@/layout/Layout";
+
+export default function About() {
+    const { t } = useTranslation("about");
+
+    return (
+        <>
+            {/* Page Intro  */}
+            <div className=' container mt-8 '>
+                <div className='flex-wrap nowrap md:text-start  text-center justify-center items-center sm:text-center'>
```

Comment:
(flex-wrap, nowrap) at the same time?
preferred to start by (text-center) then (md:text-start) to make it mobile-first approach.
when saying (text-center) => it will be applyed from (0 screen device to the next screen device mentioned,i.e. in our case (text-center md:text-start) will make the text centered from the smallest screen size up to medium screen size.